infrastructure-playbooks/purge-rgw-loadbalancer.yml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
why not including this in existing purge playbooks?

Reply:
@guits Emmm, that's a good idea, I shall put it into the `purge-cluster.yml` playbook, thanks.